src/TestUtils/src/DeviceTests/AssertionExtensions.Windows.cs

```diff
@@ -187,19 +199,26 @@ void OnViewUnloaded(object sender, RoutedEventArgs e)
 		}
 
 		public static Task<CanvasBitmap> ToBitmap(this FrameworkElement view) =>
-			view.AttachAndRun(async () =>
+			view.AttachAndRun(async (window) =>
 			{
 				if (view.Parent is Border wrapper)
 					view = wrapper;
 
+				var device = CanvasDevice.GetSharedDevice();
+
+				// HELP?
+				// The simple act of doing a window capture results in the next render method
+				// working on DirectX controls (such as Win2D).
+				// We could use this window bitmap directly, but that is extra effort to crop
+				// to the view bounds... so until this breaks...
+				using var windowBitmap = await CaptureHelper.RenderAsync(window, device);
```

Comment:
This is the magical line of code that somehow sets up the `RenderTargetBitmap.RenderAsync(view)` for success. Maybe it somehow triggers a thing in some cache...